File: 2.coding-level-one/7.fileIO/cpp/textfileio.cpp
#include <iostream>
#include <fstream>
#include <ctime>
#include <cstdlib>
using namespace std;

void readWrite(string infile, string outfile) {
	ifstream fin(infile);
	ofstream fout(outfile);
	string line;
	while(getline(fin, line)) {
		cout << line << "\n";
		fout << line << "\n";
	}
	fin.close();
	fout.close();
}
int main() {
	string infile, outfile;
	cin >> infile;
	cin >> outfile;
	readWrite(infile, outfile);
}

File: 2.coding-level-one/7.fileIO/java/TextFileIO.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.FileReader;
import java.io.FileWriter;
import java.util.Scanner;

public class TextFileIO {

	public static void readWrite(String infile, String outfile) throws Exception {
		BufferedReader fin = new BufferedReader(new FileReader(infile));
		BufferedWriter fout = new BufferedWriter(new FileWriter(outfile));
		String line;
		while ((line = fin.readLine()) != null) {
			fout.write(line);
			fout.write("\n");
		}
		fin.close();
		fout.close();
	}

	public static void main(String[] args) throws Exception {
		Scanner scanner = new Scanner(System.in);
		String infile = scanner.nextLine();
		String outfile = scanner.nextLine();
		readWrite(infile, outfile);
	}
}
